angular/tslint.json

```diff
@@ -89,7 +89,7 @@
     "no-submodule-imports": false,
     "no-switch-case-fall-through": true,
     "no-trailing-whitespace": true,
-    "no-unused-expression": true,
+    "no-unused-expression": false,
```

Comment:
Why unset this setting?

If there are variables, etc, that are completely unused, its best to remove them so they don't  clutter the code unnecessarily.

Reply:
This was causing short circuiting to fail lint

Reply:
Apparently they added a setting to account for this: https://palantir.github.io/tslint/rules/no-unused-expression/

So we just need to change it to:

```"no-unused-expression": [true, "allow-fast-null-checks"]```